change_log/next/add-ability-to-clear-dropdown-filter.yml

```diff
@@ -0,0 +1 @@
+New Features: "DropdownFilter/Ajax improvement: When clearing the dropdown in full, it now clears the selection."
```

Comment:
forward slash in the name is out of place

Reply:
This was done as this impacts `DropdownFilter` and `DropdownFilterAjax`

Reply:
I'd write both names out in order to help someone searching for changes to a particular component